docs: specify ApplicationSet can be installed in any namespace (#27579)
argoproj/argo-cd: docs/operator-manual/applicationset/Argo-CD-Integration.md

> **Use the Argo CD namespace**
>
> All ApplicationSet resources and the ApplicationSet controller must be installed in the same namespace as Argo CD. 

> > ApplicationSet resources in a different namespace will be ignored, except you enable [the feature](Appset-Any-Namespace.md).

Reviewer: ```suggestion
> ApplicationSet resources in a different namespace will be ignored, except if you enable [ApplicationSet in any namespace](Appset-Any-Namespace.md).
```

Reviewer 2: > ApplicationSet resources in different namespaces will be ignored, unless [AppSet in any namespace](Appset-Any-Namespace.md) is enabled. 

This would be the correct way to phrase it.

> > ApplicationSet resources in a different namespace will be ignored, except you enable [the feature](Appset-Any-Namespace.md).

Reviewer 2: ```suggestion
> ApplicationSet resources in different namespaces will be ignored, unless [AppSet in any namespace](Appset-Any-Namespace.md) is enabled. 
```
